.github/workflows/analysis.yml

```diff
@@ -0,0 +1,37 @@
+name: Dart CI
+
+on: [push, pull_request]
+
+jobs:
+  generator_analysis:
+    runs-on: ubuntu-latest
+    container:
+      image:  google/dart:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        working-directory: bin/objectbox_model_generator
+        run: pub get
```

Comment:
are these really necessary?

Reply:
I've removed them but it seems like analysis doesn't call pub get on the package before hand.

Reply:
Ok I'm wrong on that, I'll keep looking at the action logs